packages/tools/client-debugger/client-debugger/src/Features.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Describes features supported by {@link FluidDevtools}.
+ *
+ * @public
+ */
+export enum DevtoolsFeature {
+	/**
+	 * Indicates that the {@link FluidDevtools} instance is capable of providing Fluid telemetry logs.
+	 */
+	Telemetry = "telemetry",
+}
+
+/**
+ * Describes the set of {@link DevtoolsFeature | features} supported by a {@link FluidDevtools} instance.
+ *
+ * @remarks
+ *
+ * This has two primary purposes:
+ *
+ * 1. It can be used to signal to consumers of the debugger what kinds of functionality are supported (at runtime)
+ * by a debugger instance.
+ *
+ * 2. It can be used to make backwards compatible changes easier to make.
+ * By adding a flag to this object for new features, consumers can easily verify whether or not that feature
+ * is supported by the corresponding debugger instance before attempting to use it.
```

Comment:
```suggestion
 * 1. It can be used to signal to consumers of the Devtools what kinds of functionality are supported (at runtime)
 * by a Devtools instance.
 *
 * 2. It can be used to make backwards compatible changes easier to make.
 * By adding a flag to this object for new features, consumers can easily verify whether or not that feature
 * is supported by the corresponding Devtools instance before attempting to use it.
```